SPECS/virtiofsd/virtiofsd.spec

```diff
@@ -16,6 +16,10 @@ Source0:        https://gitlab.com/virtio-fs/virtiofsd/-/archive/v%{version}/%{n
 #   tar -czf ../%{name}-v%{version}-cargo.tar.gz vendor/
 Source1:        %{name}-v%{version}-cargo.tar.gz
 Source2:        config.toml
+Source3:        %{name}-v%{version}-cargo-vendor-CVE-2023-41051.tar.gz
```

Comment:
While I like the clear naming here, I worry about how expandable it is -- like if we have a vendor tarball that covers two CVEs or something. Probably better to name it like "v2" or something.

Reply:
Renamed to virtiofsd-v1.8.0-cargo-v2.tar.gz

Reply:
Renamed to virtiofsd-v1.8.0-cargo-v2.tar.gz